app/presenters/multi_volume_work_show_presenter.rb

```diff
@@ -1,9 +1,20 @@
 class MultiVolumeWorkShowPresenter < CurationConcernsShowPresenter
   def file_presenter_class
-    ::ScannedResourceShowPresenter
+    DynamicShowPresenter.new
   end
 
   def viewing_hint
     'multi-part'
   end
+
+  class DynamicShowPresenter
+    def new(*args)
```

Comment:
Yeah - the branch is based on the argument given to #new on initialization, which isn't available in `file_presenter_class`